### PR TITLE
rpcmethods: address requested changes

### PR DESCRIPTION
**Pull request by Cynerd**
This provides consistency with rest of the API.

Implements https://github.com/silicon-heaven/shv-doc/issues/7, https://github.com/silicon-heaven/shv-doc/issues/10, https://github.com/silicon-heaven/shv-doc/issues/9

Tries to address https://github.com/silicon-heaven/shv-doc/issues/6 and https://github.com/silicon-heaven/shv-doc/issues/8

### CODE REVIEW

File: src/rpcmethods.md [hunk 1]
```diff
@@ -720,21 +715,61 @@ result is client specific.
...
-| Null       | Int   |
+| Null      | Int    |
 
 ```
 => <id:42, method:"clientId", path:".broker/currentClient">i{}
 <= <id:42>i{2:68}
 ```
 
-#### `.broker/clientAccess`
+#### `.broker/currentClient:mountPoint`
```
Comment (fvacek) [inline src/rpcmethods.md:725]: is this function necessarry if we have `.broker/currentClient:info`?

File: src/rpcmethods.md [hunk 2]
```diff
@@ -618,26 +618,25 @@ network.
...
+| Parameter     | Result                                                                                                                                |
+|---------------|---------------------------------------------------------------------------------------------------------------------------------------|
+| Int \| String | {"clientId":Int, "userName":String\|Null, "mountPoint":String\|Null, "subscriptions":[i{1:String, 2:String\|Null}, ...], ...} \| Null |
 
-The parameter can be either *Null* (no parameter), and in such case the user info
-is for the current client. Or it can be *Int* and in such case info is for
-client with matching ID. The *Null* is returned in case there is no client with
-this ID.
+The parameter can be either  *Int* and in such case info for client with
+matching ID is provided. Or it can be *String* with SHV path for which client
```
Comment (fvacek) [inline src/rpcmethods.md:632]: I'm not sure about string parameter

File: src/rpcmethods.md (hunk 1 above)
Comment (Cynerd) [reply src/rpcmethods.md:725]: My idea... the same is for `clientId`, but you said that you need it for backward compatibility.

File: src/rpcmethods.md (hunk 2 above)
Comment (Cynerd) [reply src/rpcmethods.md:632]: In my implementation it pretty much maps one of my methods I had to implement anyway. I need to have query that gives me client that should handle some path. I also of course have query that gives me client with given client ID. In the end I can use both of them here to get client info.

File: src/rpcmethods.md (hunk 1 above)
Comment (fvacek) [reply src/rpcmethods.md:725]: We can get rid of `clientId` and `mountPoint` . Fortunately, we are using undocumented methods under `currentClient` only.

**Pull request merged**